boot/zephyr/include/mcuboot_config/mcuboot_config.h

```diff
@@ -231,6 +231,12 @@
 #define MCUBOOT_MAX_IMG_SECTORS       128
 #endif
 
+#ifdef CONFIG_BOOT_SERIAL_IMG_UPLOAD_STALLING
+#define MCUBOOT_BOOT_SERIAL_IMG_UPLOAD_STALLING 1
+#define MCUBOOT_BOOT_SERIAL_IMG_UPLOAD_STALLING_TIMEOUT	\
+	(CONFIG_BOOT_SERIAL_IMG_UPLOAD_STALLING_TIMEOUT);
```

Comment:
Why does this macro have a semicolon after it? It makes the use in the code strange, because there is no semicolon (which messes up editor indenting and stuff).

Reply:
Fixed